src/timeline/InputBar.cpp

```diff
@@ -896,39 +897,54 @@ MediaUpload::startUpload()
         QByteArray ba;
         QBuffer buffer(&ba);
         buffer.open(QIODevice::WriteOnly);
-        thumbnail_.save(&buffer, "PNG");
-        auto payload = std::string(ba.data(), ba.size());
-        if (encrypt_) {
-            mtx::crypto::BinaryBuf buf;
-            std::tie(buf, thumbnailEncryptedFile) = mtx::crypto::encrypt_file(std::move(payload));
-            payload                               = mtx::crypto::to_string(buf);
+        std::string format        = QImageReader::imageFormat(source.get()).toStdString();
```

Comment:
Probably a bad idea, if you send SVGs

Reply:
Apparently setting the quality to 0 should max out the PNG compression: https://bugreports.qt.io/browse/QTBUG-43618

Reply:
https://bugreports.qt.io/browse/QTBUG-94201

Reply:
When we set the quality, we should only use PNG because setting WebP to 0 results in *very* lossy files: <https://bugreports.qt.io/browse/QTBUG-101035>.

Yes, after thinking about it, sticking to PNG is probably the best, because easiest, therefore bugfreeest. 😊

Reply:
Let's see how big the thumbs are, if there is actually some compression used. Otherwise we can still think about reducing it more by using min(oldRes/3, 800) or lower the 800x800 thumbnail rect.